### DISCUSSION TITLE
想让输入框在聚焦全选出现问题

### DISCUSSION THREAD

**Discussion by Jiangsongheng** [ant-design/ant-design#38188] [Q&A] [answered]
### 普通input添加onFocus事件，调用focus方法可以正常实现效果。有prefix或者suffix的就不行，请问可能是什么原因？
代码写法如下：
![1](https://user-images.githubusercontent.com/59638621/197441969-438df043-12ba-4c89-833f-cfde9d646bd1.png)
![2](https://user-images.githubusercontent.com/59638621/197441980-281fa67b-a9de-4b99-96d3-2e5d8a3fea2a.png)
效果如下：
![1](https://user-images.githubusercontent.com/59638621/197442014-a9aee5de-8502-4de6-b140-4d1ec1186003.gif)

**Comment by Jiangsongheng** [comment]
求大神指导!!! QAQ

**Comment by q269384828** [comment, marked as answer]
我试了试没有问题的呀

**Reply by Jiangsongheng** [reply]
谢谢，应该是修复了，试了下现在确实可以了
